### ISSUE TITLE
"TypeError: purgeCss is not a function" : Gulp error after installing gulp-purgecss version 4.1.1

### ISSUE THREAD

**Issue by brybrant** [FullHuman/purgecss#800]
My package.json:
```json
{
  "dependencies": {
    "gulp-purgecss": "^4.0.3"
  }
}
```
Installs gulp-purgecss @ 4.1.1

When I run gulp, I get the error:
```
TypeError: purgeCss is not a function
    at purgeUnusedCss (C:\users\test\example\gulpfile.js:262:12)
    at bound (node:domain:421:15)
    at runBound (node:domain:432:12)
    at asyncRunner (C:\users\test\example\node_modules\async-done\index.js:55:18)
    at processTicksAndRejections (node:internal/process/task_queues:78:11)
```

I fixed by downgrading to version 4.0.3

**Comment by Ffloriel** [comment]
Ah yes, you would need to require the package with
```js
require('gulp-purgecss').default
```

**Comment by Ffloriel** [comment]
Released 4.1.2. `require('gulp-purgecss')` is working now
